src/objects.js

```diff
@@ -1,39 +1,72 @@
 export const createPerson = (name, age) => {
-  // your code here
+  // creates an object with the given name and age properties
+  let person = {
+    name: name,
+    age: age
+  };
+
+  return (person);
+
 };
 
 export const getName = (object) => {
-  // your code here
+  // returns the name property of the object
+  return(object["name"]);
+
 };
 
 export const getProperty = (property, object) => {
-  // your code here
+  return(object[property]);
 };
 
 export const hasProperty = (property, object) => {
-  // your code here
+  // checks whether object has given property
+  return object.hasOwnProperty(property);
 };
 
 export const isOver65 = (person) => {
-  // your code here
+  // returns true if the person is aged over 65
+  if (person['age'] > 65) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
You can achieve the same result from this function in one line, as `person['age'] > 65` evaluates to a boolean because a comparison operator (`>`) has been used. Additionally, as you know the name of the property you want to retrieve and because it doesn't contain any invalid JS characters, you can access the `age` property with dot notation (recommended):
```
return person.age > 65
```